### PR TITLE
create navbar element

### PR DESCRIPTION
**Pull request by jeffjxu**
created a navbar element that can be toggled to artist or investor view

### CODE REVIEW

File: vera-frontend/package 2.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (anish-krishnan) [inline vera-frontend/package 2.json]: Why do we have a second `package.json` file? Can we append these lines to the original file?

**Review by anish-krishnan — CHANGES_REQUESTED** [review]

**Review by Brandyn-Amado — COMMENTED** [review]
Aside from Anish's comment about the second ``package.json`` file, everything looks good to me. Once that gets cleared up I can approve.

File: vera-frontend/package 2.json (hunk 1 above)
Comment (jeffjxu) [reply vera-frontend/package 2.json]: I'm not sure why that happened, I deleted the file and its working fine

Comment (anish-krishnan) [reply vera-frontend/package 2.json]: Great, thanks!

**Review by Brandyn-Amado — APPROVED** [review]
The other PR that I merged created some merge conflicts on this branch so I went ahead and resolved them. Everything looks good now!

**Review by anish-krishnan — APPROVED** [review]
Looks great :)

**Pull request merged**